src/catalog/mod.rs

```diff
@@ -85,6 +86,45 @@ impl ColumnRefId {
     }
 }
 
+impl std::fmt::Display for ColumnRefId {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        // TODO: now ignore database and schema
+        write!(f, "${}.{}", self.table_id, self.column_id)
```

Comment:
We should make ColumnRefId more generic (e.g. `Vec<String>`) so as to represent a column in subquery. e.g. `subquery.table1.x`. I'll do this probably when refactoring binder.

Reply:
I gave up taking subquery into account. It's a little complicated. 🤯